cfgov/unprocessed/apps/youth-employment-success/js/reducers/budget-reducer.js

```diff
@@ -0,0 +1,42 @@
+import keymirror from 'keymirror';
+
+const initialState = {
+  earned: '',
+  spent: ''
+};
+
+const actionTypes = keymirror( {
```

Comment:
Worth it to create a dependency here on `keymirror`? Seems like you could have:

```
const actionTypes = {
  UPDATE_EARNED: 0,
  UPDATE_SPENT: 1
};
```

Or whatever the value you need it to be. E.g. keymirror would be creating this, right?

```
const actionTypes = {
  UPDATE_EARNED: 'UPDATE_EARNED',
  UPDATE_SPENT: 'UPDATE_SPENT'
};
```

Seems like it may be clearer to set the value directly.

Reply:
Yes, I just like not having to manually type out the string, in case of typos. I thought projects had their own `node_modules`? Or do dependencies still bloat the main js?